app.js

```diff
@@ -1,4 +1,29 @@
 //  in this file create an express application - use the middle-ware built into express
 //  to serve up static files from the public directory (index.html and client.js - you 
 //  can also serve up css files from the public directory this way if you want)
-//  you need to support a '/trucks' endpoint, and a dynamic route for '/trucks:name'
\ No newline at end of file
+//  you need to support a '/trucks' endpoint, and a dynamic route for '/trucks:name'
+
+var express = require('express');
+var app = express();
+var trucks = {
+	'Athenas': 'Meditteranean', 'Beanfish': 'Asian', 'Marination': 'Hawaaian-Korean'
+};
+app.use(express.static('public'));
+
+app.get('/trucks/:name', function (request, response) {
+	var truck = request.params.name;
+	var foodType = trucks[request.params.name];
```

Comment:
Using 'foodType' as a variable name here is confusing, because you want to return a food truck in your response for the '/trucks/:name' route. Assigning `request.params.name` to the truck variable is good, but after that what you want to do is pass that value to `trucks.getTruck(truck)` and send what that function returns to the response: ` response.send(trucks.getTruck(truck));`